feat(rbac): list roles with no permission policies (#998)
janus-idp/backstage-plugins: plugins/rbac-backend/README.md

g, group:default/team_b, role:default/team_b
```

---

**NOTE**

> When adding a role in the permission policies configuration file it is adviced to associate the role with atleast one permission policy with effect `allow`

Author: `Admin user, ensure that they have read permission for `catalog-entity`` is needed for rbac UI plugin, because the create form requires catalog entity read permission to fetch the users and groups

> When adding a role in the permission policies configuration file it is advised to associate the role with at least one permission policy with the effect `allow`.

Reviewer: ```suggestion
When you add a role in the permission policies configuration file, ensure that the role is associated with at least one permission policy with the `allow` effect.
```
